modules/migration-mtc-release-notes-1-8.adoc

```diff
@@ -0,0 +1,73 @@
+
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_content-type: REFERENCE
+[id="migration-mtc-release-notes-1-8_{context}"]
+= {mtc-full} 1.8 release notes
+
+[id="new-features-and-enhancements-1-8{context}"]
+== New features and enhancements
+
+This release has the following new features and enhancements:
+
+This release has no new features and enhancements.
+
+[id="deprecated-features-1-8_{context}"]
+== Deprecated features
+
+The following features are deprecated:
+
+This release has no deprecated features.
+
+[id="resolved-issues-1-8_{context}"]
+== Resolved issues
+
+This release has the following resolved issues:
+
+.Indirect migration is stuck on backup stage
+
+In previous releases, indirect migration becomes stuck at the backup stage, due to invalid image name error.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2233097[*BZ#2233097*])
+
+.PodVolumeRestore remain In Progress keeping the migration stuck at Stage Restore
+
+In previous releases, on performing any indirect migration, the migration becomes stuck at the `Stage Restore` step, waiting for the `podvolumerestore` to be completed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2233868[*BZ#2233868*])
+
+.Migrated application unable to pull image from internal registry on target cluster
+
+In previous releases, on migrating an application to the target cluster, the migrated application fails to pull the image from the internal image registry resulting in an `application failure`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2233103[*BZ#2233103*])
+
+.Migration failing on Azure due to authorization issue
+
+In previous releases, on trying to perform a migration on an Azure cluster, backing up to Azure storage. The migration fails at the `Backup` stage. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2238974[*BZ#2238974*])
+
```

Comment:
Suggestion: "..on an Azure cluster, when backing up to Azure storage, the migration fails.." 
I don't normally suggest lengthening a sentence, but "In previous releases...backing up to Azure storage" is not a complete sentence. Maybe create two shorter sentences, rather than my suggested long one. Your instinct for shorter sentences in good, after all.

Reply:
@RichardHoch nice suggestion.